README.md

```diff
@@ -0,0 +1,31 @@
+### Using sequences from NCBI GenBank to estimate how many species there are in the fungi genus *Stereum*
+Sarah DeLong-Duhon
+
+Reference: Fryssouli, V., Zervakis, G., Polemis, E., & Typas, M. A. (2020). A global meta-analysis of ITS rDNA sequences from material belonging to the genus Ganoderma (Basidiomycota, Polyporales) including new data from selected taxa. MycoKeys, 75(December), 71–143. https://doi.org/10.3897/mycokeys.75.59872 
+##### Intro
+There are estimated to be 2.2 to 2.8 million fungi species on earth, but we have discovered only a small fraction of them (Hawksworth & Lücking 2017, Vu et al. 2014). Even large mushroom-forming fungi, which are easy to find and document, are not very well characterized. In Fryssouli et al. 2020, the authors used almost four thousand internal transcribed spacer (ITS) sequences from public genetic databases such as NCBI GenBank and UNITE to estimate species diversity in a genus of large shelf mushrooms, some of which have a long history of medicinal use. They found 80 *Ganoderma* species, 21 of which are not named or properly/fully identified, separated into three major lineages. 	
+
```

Comment:
The paragraphs below deal with the methods used in the published paper. In a typical research article, these will go into the Materials and Methods section, or briefly summarized in the first part of the Results section. Here I suggest you clearly mark them away from the Intro, which should focus on the biological question and its significance. You can use a subheading like "Figure used as template for my project".